src/main/java/org/broadinstitute/hellbender/tools/exome/Case2PoNTargetMapper.java

```diff
@@ -96,7 +96,6 @@ public RealMatrix fromPoNToCaseCounts(final RealMatrix ponCounts) {
      * @return never {@code null} a read-count collection with the row order according to the case read count target order.
      */
     public ReadCountCollection fromPoNtoCaseCountCollection(final RealMatrix counts, final List<String> countColumnNames) {
-        return new ReadCountCollection(SetUniqueList.setUniqueList(new ArrayList<>(outputTargets)),SetUniqueList.setUniqueList(new ArrayList<>(countColumnNames)),
-                fromPoNToCaseCounts(counts));
+        return new ReadCountCollection(outputTargets, countColumnNames, fromPoNToCaseCounts(counts));
```

Comment:
The `ReadCountCollection` constructor now makes a defensive copy.  Previously the constructor took a `SetUniqueList` and literally every time it was invoked the calling code had to construct a `SetUniqueList`.  Moving the copying into the constructor eliminated all of this boilerplate.